boards/arm/blueclover_plt_demo_v2_nrf52832/blueclover_plt_demo_v2_nrf52832.yaml

```diff
@@ -17,3 +17,4 @@ supported:
   - pwm
   - spi
   - watchdog
+vendor: contextualelectronics
```

Comment:
This should be `blueclover`, not `contextualelectronics` (the latter should be in `[contextualelectronics_abc.yaml](https://github.com/zephyrproject-rtos/zephyr/pull/59642/files#diff-5e45b6298a791d47d4cd71893bcfec210a5b25c59206be68d33a109f5fb96701)).